html/pfappserver/root/src/views/Configuration/networks/interfaces/_components/TheForm.vue

```diff
@@ -71,6 +71,9 @@
           {{ $i18n.t(`Since NAT is disabled, PacketFence will adjust iptables to route traffic rather than using NAT. Make sure to add the routes on the system.`) }}
         </div>
       </b-form-group>
+      <form-group-nat-dns namespace="nat_dns"
+        :column-label="$i18n.t('Use pfdns to proxy the request to the DNS server defined above when the device is registered.')"
```

Comment:
I think this needs to be in :text to be help since that will create a huge label instead of displaying help